evap/results/views.py

```diff
@@ -80,14 +79,21 @@ def course_detail(request, semester_id, course_id):
         if section.contributor is None:
             course_sections.append(section)
         else:
-            contributor_sections.setdefault(section.contributor, []).append(section)
+            contributor_sections.setdefault(section.contributor,
+                                            {'total_votes': 0, 'sections': []})['sections'].append(section)
+
+            # Sum up all Sections for this contributor.
+            # If section is not a RatingResult:
+            # Add 1 as we assume it is a TextResult or something similar that should be displayed.
+            contributor_sections[section.contributor]['total_votes'] +=\
+                sum([s.total_count if isinstance(s, RatingResult) else 1 for s in section[3]])
```

Comment:
`ResultSection` is a namedtuple, defined in `evaluations/tools.py`. please use the name instead of `section[3]`